RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFitOOTPileupCorrection.h

```diff
@@ -130,6 +130,7 @@ class PulseShapeFitOOTPileupCorrection
     double pedSig_;
     double noise_;    
     HcalTimeSlew::BiasSetting slewFlavor_;    
+    const double overlapLimit = 1.0;
```

Comment:
This first plot was the reasoning behind the dt < 1 ns. These are cases where there are two pulses directly on top of each other, where one pulse would have sufficed, and are the most obvious cases where we need to be sure we are accounting for all of the energy.